src/search/pdbs/pattern_generator_cegar.h

```diff
@@ -18,7 +18,7 @@ class PatternGeneratorCEGAR : public PatternGenerator {
     virtual PatternInformation compute_pattern(
         const std::shared_ptr<AbstractTask> &task) override;
 public:
-    explicit PatternGeneratorCEGAR(options::Options &opts);
+    explicit PatternGeneratorCEGAR(const plugins::Options &opts);
```

Comment:
Why do we add const here?

Reply:
Same as above. I'll resolve these conversations. Feel free to reopen if you disagree.